### PR TITLE
refactor: remove unused function from ExplorerAPI subclass [APE-638]

### PR DESCRIPTION
**Pull request by fubuloubu**
### What I did

depends on: https://github.com/ApeWorX/ape/pull/1305

### Checklist

- [x] Passes all linting checks (pre-commit and CI jobs)
~~- [ ] New test cases have been added and are passing~~
~~- [ ] Documentation has been updated~~
- [x] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)
- [x] Add pin on version of ape core that https://github.com/ApeWorX/ape/pull/1305 is released in

### CODE REVIEW

File: ape_etherscan/explorer.py [hunk 1]
```diff
@@ -55,21 +55,6 @@ def get_contract_type(self, address: AddressType) -> Optional[ContractType]:
 
         return contract_type
 
-    def get_account_transactions(self, address: AddressType) -> Iterator[ReceiptAPI]:
```
Comment (unparalleled-js) [inline ape_etherscan/explorer.py:58]: Maybe the tests for this could be migrated to the query tests.

Comment (fubuloubu) [reply ape_etherscan/explorer.py:58]: I don't think this had tests

**Comment by fubuloubu** [comment]
NOTE: [95932b2](https://github.com/ApeWorX/ape-etherscan/pull/72/commits/95932b2c92a20830256de42e608e3b26c3a72590) is the solution to a long-standing bug I've had with etherscan (it kept missing nonce 0 receipts)

**Review by unparalleled-js — APPROVED** [review]

**Comment by fubuloubu** [comment]
Just going to double check this one against the latest release

**Comment by fubuloubu** [comment]
Works with latest release

**Comment by unparalleled-js** [comment]
linting concerns hopefully addressed now from main branch changes

**Review by unparalleled-js — DISMISSED** [review]

**Review by unparalleled-js — APPROVED** [review]

**Pull request merged**